b2/console_tool.py

```diff
@@ -1132,7 +1136,8 @@ class Sync(Command):
     """
 
     OPTION_FLAGS = [
-        'delete', 'noProgress', 'skipNewer', 'replaceNewer', 'dryRun', 'allowEmptySource'
+        'delete', 'noProgress', 'skipNewer', 'replaceNewer', 'dryRun', 'allowEmptySource',
+        'excludeAllSymlinks'
```

Comment:
please put a `,` after the last entry and re-run yapf, it will make a multi-line statement diff-friendly